entities/entities/meteor/cl_init.lua

```diff
@@ -1,6 +1,8 @@
 ENT.Spawnable = false
 ENT.AdminSpawnable = false
 
+ENT.ParticleModel = Model("particles/smokey")
```

Comment:
This forces the caching of the model on server startup, adding to the first frame lag that's already there. I think it's better to cache it upon first use, especially since barely anyone ever uses the meteor.

Reply:
It is automatically cached on use, so removing it, would be more suitable.